mbed_app.json

```diff
@@ -479,8 +470,10 @@
             "storage.storage_type"                      : "TDB_INTERNAL"
         },
         "WIO_BG96": {
+            "events.shared-eventsize"                   : 2048,
```

Comment:
no need to repeat the events here. Its already on top-section.

Reply:
Done.